NYPL/digital-collections
# DR 2872 puppeteer script for facelift homepage

## Ticket:

- JIRA ticket [Configure passenger settings for Repo API](https://jira.nypl.org/browse/DR-2872)

## This PR does the following:

- Adds a test script to assert that swim lane images load on the homepage

## How has this been tested?

- Install puppeteer with `npm install puppeteer`
- Get on the VPN and accessinate your IP for nypl-dams-dev
- Run the test against qa with `node test/homepageImageLoadingTest.js`. If it does not time out, that means it succeeded. It should print all images loaded and the time taken for the swim lane images to finish loading, including the 12 that require scrolling down the page. This simulates a single user loading the homepage one time.
- Run the test against production with `node test/homepageImageLoadingTest.js --prod`. The above still applies.

## Accessibility concerns or updates

- None

### Checklist:

- [ x ] All new and existing tests passed.
